src/resources/js/login.js

```diff
@@ -36,7 +36,7 @@ var observeLogin = function (form, url, secureUrl) {
                 }
 
                 if (refresh) {
-                    $('#success').show();
+                    $('#success').show().css('visibility', 'visible');
```

Comment:
Switch `.hide()` on line  89 with `.css('visibility', 'hidden')`.

Then you can loose `.show()` on this line.